test/cpp/test_ir.cpp

```diff
@@ -331,5 +333,35 @@ TEST_F(IrTest, TestSizeDivNodeDynamic) {
   EXPECT_EQ(dim_node_div->getDynamicValue(), 0);
 }
 
+TEST_F(IrTest, TestSizeDivNodeDynamicByZero) {
+  torch::lazy::Value scalar_value =
+      torch::lazy::Value(ScalarOp(0.0, xla::F32), 0);
+  std::vector<int64_t> target_size = {2, 2};
+  torch::lazy::Value node = torch::lazy::Value(
+      torch::lazy::MakeNode<Expand>(scalar_value, target_size));
+  torch::lazy::NodePtr nonzero_node = torch::lazy::MakeNode<NonZero>(node);
+
+  // static value = 4, dynamic value = 0
+  torch::lazy::NodePtr size_node_nonzero_0 =
+      torch::lazy::MakeNode<SizeNode>(nonzero_node, 0);
+  std::shared_ptr<torch::lazy::DimensionNode> dim_size_node_nonzero_0 =
```

Comment:
this is ded or were you going to use it?

Reply:
good catch.